### PR TITLE
Add Numi to Utilities

### PR DESCRIPTION
**Pull request by brunowego**
:+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@
 - [Keka](http://www.kekaosx.com/en/) - Compress to and extract from many archive file formats. ![Open-Source Software][OSS Icon]
 - [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. ![Open-Source Software][OSS Icon]
 - [Monodraw](http://monodraw.helftone.com/) - A powerful ASCII art editor.
+- [Numi](http://numi.io/) - Beautiful calculator app for Mac.
```
Comment (iCHAIT) [inline README.md]: Description can be `Beautiful calculator app` no need to use `for mac`.

**Comment by iCHAIT** [comment]
The UI is so clean, like this, thanks for sharing.

**Comment by brunowego** [comment]
Done.

**Comment by iCHAIT** [comment]
:+1:

**Pull request merged**